Base/QTGUI/qSlicerStyle.cxx

```diff
@@ -127,6 +127,11 @@ int qSlicerStyle::pixelMetric(PixelMetric metric, const QStyleOption * option,
     case QStyle::PM_SliderLength:
       return 12; // default to 27
       break;
+#if (QT_VERSION > QT_VERSION_CHECK(5, 0, 0))
```

Comment:
For sake of correctness, this would be `if (QT_VERSION >= QT_VERSION_CHECK(5, 0, 0))`

Reply:
`>=` instead of `>`